### PR TITLE
Recording

### PR DESCRIPTION
**Pull request by HasaniH**
Very coupled firebase and business logic, but functional app

please review @nyghtowl

### CODE REVIEW

File: android/cloud_spin_v2/app/build.gradle [hunk 1]
```diff
@@ -1,50 +1,31 @@
-// Copyright 2017 Google Inc.
```
Comment (mimming) [inline android/cloud_spin_v2/app/build.gradle]: Re-add the license header

File: android/cloud_spin_v2/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -25,4 +30,14 @@ Copyright 2017 Google Inc.
     <string name="status_uploading">Uploading</string>
     <string name="status_uploaded">Uploaded</string>
     <string name="status_failed">Failed</string>
+
+    <string name="intro_message">
+        <![CDATA[
+
+
+            This sample demonstrates how to record video using Camera2 API.
```
Comment (mimming) [inline android/cloud_spin_v2/app/src/main/res/values/strings.xml]: is this field used?  Does it need to be a multiline field?

File: android/cloudspincontroller/app/src/main/java/com/google/cloudspincontroller/MainActivity.java [hunk 3]
```diff
@@ -43,25 +46,73 @@
 
 public class MainActivity extends AppCompatActivity {
 
+    //Constants
```
Comment (mimming) [inline android/cloudspincontroller/app/src/main/java/com/google/cloudspincontroller/MainActivity.java]: Space before comments

File: android/cloud_spin_v2/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (HasaniH) [inline android/cloud_spin_v2/app/src/main/res/values/strings.xml]: Yes, it's used on the information button clicked. Doesn't need to be a multiline field however. I'll resolve that

**Pull request merged**